src/gitea/pull_request/base.py

```
@@ -0,0 +1,78 @@
+
+        Args:
+            owner: The owner of the repository.
+            repository: The name of the repository.
+
+        Returns:
+            The endpoint URL for listing pull requests.
+
+        """
+        return f"/repos/{owner}/{repository}/pulls"
+
+    def _list_pull_requests_helper(  # noqa: PLR0913
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)